inference_perf/metrics/prometheus_client.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2025 The Kubernetes Authors.
```

Comment:
Can this be moved to it own package directory `meritcs/prometheus`? This may help organization.